include/seqan/arg_parse/arg_parse_type_support.h

```diff
@@ -71,6 +93,23 @@ inline TTarget _cast(TString const s)
 }
 
 // ----------------------------------------------------------------------------
+// Function _castBool()
+// ----------------------------------------------------------------------------
+
+template <typename TString>
+inline bool _castBool(bool & dst, TString const s)
+{
+    // since the validValue check already verifies that the value if one of
```

Comment:
There is no possibility do to this here. Every user input is checked automatically if it's value can be found in the (non empty) `option._validValues` list (look [here](https://github.com/seqan/seqan/blob/master/include/seqan/arg_parse/arg_parse_argument.h#L764) ). One would have to add a special case only for boolean values which I think is not consistent with the general use of the argument parser.

If the user input is not inside the valid values the argument parser will indicate such.
```
myapp: the given value 'off' is not in the list of allowed values [1, ON, TRUE, T, YES, 0, OFF, FALSE, F, NO]
```